### PR TITLE
new pipelines 1.13 changes

### PR DESCRIPTION
**Pull request by rcerven**
# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] Python type annotations added to new code
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

**Review by chmeliik — APPROVED** [review]

**Review by mkosiarc — APPROVED** [review]

File: atomic_reactor/plugins/gather_builds_metadata.py [hunk 1]
```diff
@@ -110,7 +111,7 @@ def _get_build_metadata(self, platform: str):
         """
         pullspec_image = self._determine_image_pullspec(platform)
         buildroot = get_buildroot(platform)
-        build_host = self._get_hostname_for_platform(platform)
+        build_host = json.loads(self._get_hostname_for_platform(platform))
```
Comment (chmeliik) [inline atomic_reactor/plugins/gather_builds_metadata.py:114]: Why is this change needed?
